gradle.properties

```diff
@@ -13,15 +13,15 @@
 #
 
 mavenCentralMirror=https://mirrors.cloud.tencent.com/nexus/repository/maven-public/
```

Comment:
Gradle does not have a built-in Repo Mirror setting like Maven does, we have to explicitly expose a variable. We can keep it with the default value `https://repo1.maven.org/maven2/`. Users can override this variable in `$HOME/.gradle/gradle.properties`